language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/resolvers/parsercontext/ParserRuleConstantDefinitionContextResolver.java

```diff
@@ -18,40 +18,19 @@
 
 package org.ballerinalang.langserver.completions.resolvers.parsercontext;
 
-import org.ballerinalang.langserver.DocumentServiceKeys;
 import org.ballerinalang.langserver.TextDocumentServiceContext;
-import org.ballerinalang.langserver.completions.PossibleToken;
-import org.ballerinalang.langserver.completions.consts.ItemResolverConstants;
-import org.ballerinalang.langserver.completions.consts.Priority;
 import org.ballerinalang.langserver.completions.resolvers.AbstractItemResolver;
 import org.eclipse.lsp4j.CompletionItem;
 
 import java.util.ArrayList;
-import java.util.List;
 
 /**
  * constant definition context resolver for the completion items.
  */
 public class ParserRuleConstantDefinitionContextResolver extends AbstractItemResolver {
```

Comment:
Do we need this class? it only returns an empty list.

Reply:
Yes, this is to maintain the backward compatibility since this particular context based item resolving need a separate revamp